python/.gitignore

```diff
@@ -24,6 +24,7 @@ typings/
 
 # Config secrets
 config/config.js
+config.ini
```

Comment:
Does this file do anything? ie a `.gitignore` not located at the git repo root directory?

Reply:
I don't know, this was added by @iam-robi, maybe it's to run standalone in a docker container

Reply:
yes I used a gitignore for this directory as its opened by pycharm on my side + it tends to outputs some formats and big files like .csv (i should probably add rules like *.csv). 
I tend to use .gitignore in subfolders when rules are specific to the context of the folder like here __pycache__ is very python related. I can update it to be drier or move it to the main gitignore if needed